packages/typography/src/Typography.tsx

```diff
@@ -302,12 +341,15 @@ function Disclaimer({ children, className, ...rest }: DisclaimerProps) {
 
 Disclaimer.displayName = 'Disclaimer';
 
+/**
+ * Overline
+ */
 const overline = css`
   font-size: 12px;
-  font-weight: 600;
+  font-weight: bold;
```

Comment:
This is trivial but should we stick to using numbers since we defined numbers in the font faces?